starter-code/MyLinkedList.java

```diff
@@ -13,50 +13,118 @@ public int getSize() {
 
 	public boolean remove(int index) {
 		//to-do
+
+		/*
+		head
+		 */
+		if (index == 0) {
+
+			head = head.getNext();
+
+		/*
+		tail
+		 */
+		} else if (index +1 == size) {
+
+			getNodeAt(index - 1).setNext(null);
+
+		/*
+		previous node
+		*/
+		} else {
+
+			Node<T> previousNode = getNodeAt(index -1);
+			Node<T> nextNode = previousNode.getNext().getNext();
+
+			previousNode.setNext(nextNode);
+		}
```

Comment:
Don't forget the case where `index` is a bogus value, like a negative number. You can just return `false` in that case. Also, don't forget to return `true` when you do remove something!